app/renderer/components/about.js

```diff
@@ -12,19 +11,24 @@ module.exports = {
         };
     },
     template: `
-    <div id="about-modal" class="modal">
-      <div class="modal-content">
-        <h4>Gaucho</h4>
-        <p>Version: {{version}} </p>
-        <i>by @angrykoala</i>
-        <p>Gaucho is Open Source software licensed under GNU GPL V3, it can be downloaded for free at:</br><a v-on:click="openLink" href="#">https://github.com/angrykoala/gaucho</a></p>
-      </div>
-      <div class="modal-footer">
-        <a href="#!" class="modal-action modal-close waves-effect waves-green btn-flat">Close</a>
-      </div>
-    </div>
+      <a v-on:click="openAbout">About</a>
     `,
     methods: {
+        openAbout() {
+            AppAlert.toggle(
+                '<h4>Gaucho</h4>',
```

Comment:
So, my idea was (and I'm still not sure if it would work), instead of only having a link in the about template, put all this html code as part of the template, and call AppAlert in Navbar.

Something like this:
```js
AppAlert.toggle(html: `<about></about>`);
```
_navbar.js_

This way, the html will be an actual component instead of a plain string in a method, and the about component will be something more than just a link

thought on that? If you don't like it or it is a bit hard to achieve I'm ok with using you current solution

Reply:
i though the above and already tried but it doesnt work because it doesn't get translated. That property expects plain html. Do you have any suggestion if this can be skipped somehow?

Reply:
Ok, I though it may have worked because you were using v-on attributes.

I think everything is correct, I will try and see if I can improve that somehow. I', going to check that everything is working and I'll merge afterwards